packages/core/src/controller/base-controller.ts

```diff
@@ -27,15 +27,14 @@ abstract class BaseController implements interfaces.Controller {
      * @param successStatusCode - The HTTP status code to return upon successful execution.
      */
     protected async callUseCaseAsync(useCase: Promise<any>, res: any, successStatusCode: number) {
-
-        let dataReturn: any;
-
         try {
-            dataReturn = await useCase;
-
+            const dataReturn = await useCase;
             return res.status(successStatusCode).json(dataReturn);
         } catch (error: any) {
-            Report.Error(error, undefined, this.serviceName);
+            if (error instanceof Error) {
```

Comment:
Please @f0ntana verify if you throw/throw the default error from node will create the same weird error structure as such:
`statusCode: 500 { // this line can't exist in any sense
     statusCode: 400,
     errorMessage: "any error",
     stack
}
`

Reply:
<img width="1567" alt="Print2" src="https://github.com/expressots/expressots/assets/6570783/a1705b9f-15a2-4338-9a17-8f816892da5a">
<img width="1575" alt="Print1" src="https://github.com/expressots/expressots/assets/6570783/7aec5900-0c80-473a-bb92-ced31e6d6a59">
<img width="1547" alt="Print3" src="https://github.com/expressots/expressots/assets/6570783/e26d4db9-9656-4834-89c5-f19508cba505">

Reply:
@rsaz did I provide all the cases? or do we need more?